doc/transform_methods/hansenlaw.rst

```diff
@@ -121,6 +121,19 @@ If you would like to access the Hansen-Law algorithm directly (to transform a
 right-side half-image), you can use :func:`abel.hansenlaw.hansenlaw_transform`.
 
 
+
+Tips
+----
+
+`hansenlaw` tends to perform better with images of large size :math:`n \gt 1001` pixel width. For smaller images the angular_integration (speed) profile may look better if sub-pixel sampling is used via: ::
+
+    transform_options=dict(dr=0.1)
```

Comment:
I think that your re-binning example is actually a great example of how `dr` can be used. In your case, the actual camera pixels are a real-world unit. By including the appropriate `dr` to connect the size of the pixels in your image with the real camera pixels, you correctly rescale the image. Alternatively, your rescaling algorithm should take into account the change in pixel size and then simple call PyAbel with dr=1.

In any case, I still think that you should remove the suggestion about 

`transform_options=dict(dr=0.1)`

I think that the only relevant parameter for improving the quality of the angular integration should be to set 

`angular_integration_options=dict(dr=0.1)`

Also, I'm a little bit surprised that 0.1 pixel sampling is required. I would have thought that 0.5 or 0.2 pixel sampling would be more than sufficient to improve the quality of the angular integration, but you have experimented with this much more than I have.